### PR TITLE
Add ios colours.

### PR DESCRIPTION
**Pull request by bwinton**
Note: We'll need an extra convenience function in `Shared/Extensions/UIColorExtensions.swift` that looks like:
``` swift
    /**
     * Initializes and returns a color object for the given RGBA hex integer.
     */
    public convenience init(rgb: Int) {
        self.init(
            red: CGFloat((rgb & 0xFF0000) >> 24) / 255.0,
            green: CGFloat((rgb & 0xFF0000) >> 16) / 255.0,
            blue: CGFloat((rgb & 0x00FF00) >> 8)  / 255.0,
            alpha: CGFloat((rgb & 0x0000FF) >> 0)  / 255.0)
    }
```

### CODE REVIEW

File: photon-colors/build.js [hunk 1]
```diff
@@ -61,52 +64,54 @@ const formats = {
...
+        return `${r} ${g} ${b} ${color} ${variant}\n`;
       }
     },
-    'ext': 'less'
+    'ext': 'gpl'
   },
-  'sass': {
-    'output': [`/* Photon Colors SCSS Variables v${metadata.version} */\n\n`],
+  'ios': {
+    'output': [`/* Photon Colors JS Variables v${metadata.version}\n   From ${metadata.homepage} */\n\nextension UIColor {\n    struct Photon {\n`],
```
Comment (aminalhazwani) [inline photon-colors/build.js]: The output should probably be "Photon Colors iOS Variables..." instead of "Photon Colors JS Variables..." 🙂

Comment (aminalhazwani) [reply photon-colors/build.js]: Same line, is it necessary to add the homepage address here?

**Review by aminalhazwani — CHANGES_REQUESTED** [review]
Hey @bwinton it looks good to me! I just found a small typo that needs to be fixed. I have no way to actually test the `.swift` palette. Shall we reach out to an iOS engineer maybe? Or did you already? 🙂 

I also have a question regarding the Mozilla license. While doing the review I noticed that sometimes we include it in the output and other times we do not. Is this something that depends on the platform or is this something that we should do for every output file? 

Another thing that I noticed only now, the colors are not alphabetically ordered. Is this something we want to pursue?

We should also bump the version to 2.3.0 in the `package.json` (or 3.0.0 if we do more changes like the ones mentioned above).

**Comment by aminalhazwani** [comment]
Regarding the license I see that the current `UIconstant.swift` has it at the top https://github.com/mozilla-mobile/firefox-ios/blob/master/Client/Frontend/UIConstants.swift#L1. 

Another question regarding the implementation of the iOS variables. How would the engineers like to import them? Via NPM (I see that they have a `package.json` https://github.com/mozilla-mobile/firefox-ios/blob/master/package.json) or would they prefer to have a `Pod` package? Like [this example](https://github.com/mozilla-mobile/firefox-ios/blob/master/package.json) from the Salesforce design system.

File: photon-colors/build.js (hunk 1 above)
Comment (bwinton) [reply photon-colors/build.js]: Fixed, and I think it is.

**Comment by bwinton** [comment]
* I reached out to Farhan, who gave a 👍 to the changes.
* I added the license. I think we should probably add it everywhere we can, so I did.
* I think we want to leave the colours in the order they are in the JSON, which is hopefully the order we want them in the Design System, for ease of doing that. 😉
* And I bumped it to 3.0.

For importing, Farhan thinks that just copy-pasting is the best solution. Pods (well, Carthage packages) are way too heavyweight, and they don't use npm for building (just for config changes). It's not ideal, but I think it's fine for now.

**Review by aminalhazwani — APPROVED** [review]

**Comment by aminalhazwani** [comment]
This is great! Thank you @bwinton! I am working rn on the `photon-colors/README` file so that we can publish it together when we update the NPM module to v3.0.0!

**Pull request merged**